WEBSITE_CONTRIBUTING.md

```diff
@@ -11,6 +11,77 @@ Contributing to the Podman Desktop website is a valuable way to support the proj
 
 The below information outlines details on how to both contribute as well as our documentation "templates" and criteria.
 
+## Getting Started
+
+### Install the project dependencies
+
+```shell-session
+$ pnpm install
+```
+
+### Local Development of the website / documentation
+
+```shell-session
+$ pnpm start
+```
+
+This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+
+You may also run:
+
+```sh
+pnpm website:dev
+```
+
+Which will automatically open your browser to `localhost:3000`.
```

Comment:
The above information is already covered in the contribution guidelines [here](https://github.com/podman-desktop/podman-desktop/blob/main/WEBSITE_CONTRIBUTING.md#previewing-the-website). So, no need to introduce another section altogether to avoid redundancy.
As far installing dependencies are concerned, all the relevant content is already present [here](https://github.com/podman-desktop/podman-desktop/blob/main/WEBSITE_CONTRIBUTING.md#previewing-the-website).